page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/fragments/FragmentReplacer.java

```diff
@@ -0,0 +1,127 @@
+package ru.sbtqa.tag.pagefactory.fragments;
+
+import com.google.common.collect.ImmutableMap;
+import cucumber.runtime.model.CucumberFeature;
+import gherkin.ast.Feature;
+import gherkin.ast.GherkinDocument;
+import gherkin.ast.ScenarioDefinition;
+import gherkin.ast.Step;
+import gherkin.ast.Tag;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import ru.sbtqa.tag.pagefactory.exceptions.FragmentException;
+import ru.sbtqa.tag.pagefactory.utils.ReflectionUtils;
+
+public class FragmentReplacer {
+
+    private static final String FRAGMENT_TAG = "@fragment";
+    //TODO need to get i18n
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=63&issues=AWSSY2K4sk71k7lwrek7&open=AWSSY2K4sk71k7lwrek7)

[ISSUE_KEY]: AWSSY2K4sk71k7lwrek7